src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.lock.json

```diff
@@ -4,10 +4,78 @@
   "targets": {
     "DNXCore,Version=v5.0": {
       "coveralls.io/1.4": {},
-      "Microsoft.DotNet.PerfTools/0.0.1-prerelease-00077": {},
-      "Microsoft.NETCore.Runtime/1.0.1-beta-23213": {},
-      "Microsoft.NETCore.TestHost-x64/1.0.0-beta-23213": {},
-      "Microsoft.NETCore.TestHost-x86/1.0.0-beta-23213": {},
+      "Microsoft.DotNet.xunit.performance.analysis/1.0.0-alpha-build0012": {},
+      "Microsoft.DotNet.xunit.performance.runner.Windows/1.0.0-alpha-build0012": {
+        "dependencies": {
+          "xunit.runner.console": "2.1.0-beta4-build3109"
```

Comment:
For some reason this keeps getting locked to this version, even though the dependency listed in Microsoft.DotNet.xunit.performance.analysis/1.0.0-alpha-build0012 is actually beta3. I have no idea why that's happening. Regardless, we aren't actually using this dependency anywhere (it's the desktop console runner), so it is fairly harmless. We may just want to remove this dependency, as you don't need to use the desktop console runner with the performance runner.